build.sc

```diff
@@ -621,6 +623,25 @@ trait Cli extends SbtModule with ProtoBuildModule with CliLaunchers
   }
   def generatedSources = super.generatedSources() ++ Seq(constantsFile())
 
+  def defaultFilesResources = T.persistent {
+    val dir = T.dest / "resources"
+    val resources = Seq(
+      "https://raw.githubusercontent.com/scala-cli/default-workflow/main/.github/workflows/ci.yml" -> (os.sub / "workflows" / "default.yml"),
```

Comment:
I see that definition of workflows contains:
```
- uses: coursier/setup-action@v1.2.0-M3
      with:
        jvm: temurin:17
```
which install scala-cli, so do we still need the `scala-cli-setup`? maybe Scala CLI should only promote `coursier/setup-action@v1.2.0-M3` to install `scala-cli`.